comp/process/agent/agent_linux.go

```diff
@@ -51,11 +51,10 @@ func Enabled(config config.Component, checkComponents []types.CheckComponent, lo
 	case flavor.DefaultAgent:
 		if npmEnabled {
 			if runInCoreAgent {
-				log.Error("Network Performance Monitoring is not supported in the core agent. " +
-					"Please ensure the process-agent is enabled as a standalone agent to collect " +
-					"process, container and network performance metrics.")
+				log.Info("Network Performance Monitoring is not supported in the core agent. " +
+					"The process-agent will be enabled as a standalone agent to collect network performance metrics.")
 			}
-			return false
+			return true
```

Comment:
Nit: this could be simplified as
```go
if npmEnabled && runInCoreAgent {
	log.Info(...)
}
return runInCoreAgent
```

Can be handled in a follow-up though, I don't think we should re-run the CI just for this change